index.html

```diff
@@ -237,35 +237,35 @@ <h3 class="center">Contributors</h3>
         <ul class="normalize-ul center">
           <li>
             <img class="profile-picture" src="https://github.com/matthiasliszt.png" alt="Matthias Liszt" />
-            <a href="https://matthiasliszt.github.io">Matthias Liszt</a>
+            <a href="https://matthiasliszt.github.io" target="_blank" rel="noopener">Matthias Liszt</a>
```

Comment:
Could you please also add noreferrer; `rel="noopener noreferrer"`?

Reply:
I actually realised after creating the commit that no other links on the page has the target=_blank attribute and thought that there might be a reason so I can remove these ones instead to have the same behaviour on all links (or add the target=_blank on all links, either way)

Reply:
Disregard my previous comment. I misunderstood. Leave as is.